app/views/videos/show.html.haml

```diff
@@ -7,4 +8,35 @@
         %header
           %h3= @video.title # No need for string interpolation as the expression returns a string anyways
           %span Rating: 4.5/5.0
-        %p= @video.description
\ No newline at end of file
+        %p= @video.description
+        .actions
+          %a.btn.btn-primary(href="") Watch Now
+          %a.btn.btn-default(href="") + My Queue
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for [@video, @review] do |f|
+        %fieldset
+          .row
+            .col-sm-3
+              = f.form_group do
+                = f.select :rating, [[5, "5 stars"], [4, "4 stars"], [3, "3 stars"], [2, "2 stars"], [1, "1 stars"]], { label: "Rate this video" }, { class: "form-control", control_col: "col-sm-3"}
+          .row
+            .col-sm-8
+              = f.form_group do
+                = f.text_area :comment, label: "Write Review", control_col: "col-sm-8", rows: "6"
+        %fieldset.form-group.actions.clearfix
+          = f.submit "Submit"
+          = link_to "Cancel", @video
+      %header
+        %h3 User Reviews (#{@video.reload.reviews.count})
```

Comment:
This `reload` belongs in the controller, not in the view. Keep as much "logic" out of the view as possible. What you have actually done here is caused three database hits. In the controller, line `33` and line `35`. I'm not actually sure how much of a performance issue this is, but it is bad form.